src/chainRec.js

```diff
@@ -0,0 +1,35 @@
+var _arrayChainRec = require('./internal/_arrayChainRec');
+var _curry3 = require('./internal/_curry3');
+
+
+/**
+ * `chainRec` does stack safe recursion on List
```

Comment:
It doesn't look like we use capital "List" in our descriptions, just for the category.. could you change this to "on lists."?